client/lib/reader-lists/actions.js

```
@@ -123,18 +130,50 @@ var ReaderListActions = {
 		}
 
 		Dispatcher.handleViewAction( {
-			type: 'CREATE_READER_LIST',
+			type: actionTypes.CREATE_READER_LIST,
 			data: { title: title }
 		} );
 
 		wpcom.undocumented().readListsNew( title, function( error, data ) {
 			Dispatcher.handleServerAction( {
-				type: 'RECEIVE_CREATE_READER_LIST',
+				type: actionTypes.RECEIVE_CREATE_READER_LIST,
+				data: data,
+				error: error
+			} );
+		} );
+	},
+
+	update: function( owner, slug, title, description ) {
+		if ( ! owner || ! slug || ! title ) {
+			return;
```

maybe throw an Error here instead? at least in dev.